<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="agregar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(2, 2, 2);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:0.339795 rgba(255, 0, 0, 255), stop:0.339799 rgba(255, 255, 255, 255), stop:0.662444 rgba(255, 255, 255, 255), stop:0.662469 rgba(0, 0, 255, 255), stop:1 rgba(0, 0, 255, 255));
background-color: rgb(144, 167, 200);</string>
    </property>
    <widget class="QLabel" name="labelEmail">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUsuario">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>180</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Agregue Correo</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPassword">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>contraseña</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUsuario">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGuardar">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>340</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    background-color: rgb(255, 255, 255);
    border-radius: 15px;
    border: 1px solid rgb(144, 167, 200);
}

QPushButton:hover{
    background-color: rgb(227, 227, 227);
    border: 1px solid rgb(120, 140, 180);
}</string>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
    <widget class="QLabel" name="regresar">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>320</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>
        QLabel {
            font-family: &quot;Arial Rounded MT Bold&quot;;
            font-size: 16pt;
            border-radius: 10px;
        }
        QLabel:hover {
            background-color: #ccc;
        }
        QLabel:pressed {
            background-color: #aaa;
        }
     </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../flecha.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEmail">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>130</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Agregue usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>230</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Agregue contraseña</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
